accesslog/middleware.go

```diff
@@ -56,8 +59,16 @@ func (mw *AccessLogMiddleware) MiddlewareFunc(h rest.HandlerFunc) rest.HandlerFu
 
 		logger := requestlog.GetRequestLogger(r)
 		util := &accessLogUtil{w, r}
-
-		logger.Print(mw.executeTextTemplate(util))
+		logger.WithFields(logrus.Fields{
+			"type":         TypeHTTP,
+			"ts":           util.StartTime().Round(0),
+			"status":       util.StatusCode(),
+			"responsetime": util.ResponseTime().Seconds(),
```

Comment:
just nitpicking: would be nice to have here `reposnsetimeseconds` or anything to indicate that these are seconds.

Reply:
I prefer to keep it as `responsetime` because `responsetimeseconds` is way too long, if you don't mind. We can assume second is standard unit of measure for time, I think it is pretty save and conventional.

Reply:
nessun problema!